### PR TITLE
Sign in instructions email for social network users

### PR DESCRIPTION
**Pull request by karenc**

### CODE REVIEW

File: app/views/sign_in_help_mailer/sign_in_help.text.erb [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
 
+<% else %>
+
+<% social_auths = @user.authentications.reject { |a| a.provider == 'identity' } %>
+<% if social_auths.present? %>
+You have previously signed in with
+
+<% social_auths.each do |social_auth| %>
+  - Sign in with <%= social_auth.provider.capitalize %>
+    <%= root_url %>auth/<%= social_auth.provider %>
```
Comment (jpslav) [inline app/views/sign_in_help_mailer/sign_in_help.text.erb]: Can you make this a `link_to`?

Comment (jpslav) [inline app/views/sign_in_help_mailer/sign_in_help.text.erb]: e.g. `<%= link_to social_auth.provider, "#{root_url}/auth/#{social_auth.provider}" %>`

Comment (karenc) [inline app/views/sign_in_help_mailer/sign_in_help.text.erb]: This is supposed to be the text email though.

Comment (jpslav) [inline app/views/sign_in_help_mailer/sign_in_help.text.erb]: Agreed.

**Pull request closed without merging**